docs/content/concepts/metadata-tags.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: "Metadata & tags | Dagster Docs"
+description: "Dagster offers several ways to provide useful information and documentation alongside your data pipelines, including metadata and tagging."
+---
+
+# Metadata & tags
+
+One of the core benefits of Dagster is fostering collaboration between the engineers who build data pipelines and the end users who consume the data pipelines produce.
+
+Dagster offers several ways to provide useful information and documentation alongside your data pipelines, including metadata and tagging.
+
+---
+
+## Benefits
+
+Using metadata and tags helps you:
+
+- Create built-in documentation that makes your pipelines easy to understand
+- Provide useful context for other users of your project (and your future self!)
+- Improve the ease of debugging when issues arise
+- Organize definitions in your Dagster project and improve filtering in the Dagster UI
+
+---
+
+## How metadata works
+
+There are two main types of metadata in Dagster: definition and runtime. In code, you can attach metadata to definitions like assets, jobs, and runs. You'll then be able to view the metadata in the Dagster UI.
+
+### Definition metadata
+
+**Definition metadata** is information that's fixed or doesn't frequently change. For example, definition metadata could be:
+
+- A description of an asset
```

Comment:
Description and group are metadata in a broad sense, but not in the strict sense that I think we should be using the term within this doc.  To be specific about what I mean here, when you define an asset, there are top-level fields you can set like `description`, `group_name`, and `metadata`.  `metadata` is basically lets users store arbitrary information that helps provide context to about asset, but isn't captured in Dagster's core data model.

Some examples of definition metadata:
- The columns an asset is expected to contain
- A link to source file, in Github, that contains the code that defines the asset or job